packages/apis/src/verifier_simple.rs

```diff
@@ -95,6 +95,29 @@ pub struct OperatorVoteInfoResponse {
     pub result: String,
 }
 
+/// Metadata for a task - indexed by (task_queue, task_id)
+#[cw_serde]
+pub struct TaskMetadata {
+    pub power_required: Uint128,
+    pub status: TaskStatus,
+    pub created_height: u64,
+    /// Measured in UNIX seconds
+    pub expires_time: u64,
+}
+
+impl TaskMetadata {
+    pub fn is_expired(&self, env: &Env) -> bool {
+        env.block.time.seconds() >= self.expires_time
+    }
+}
+
+/// Metadata for a given vote by an operator - indexed by (task_queue, task_id, operator)
+#[cw_serde]
+pub struct OperatorVote {
```

Comment:
This should not be common, as I want to allow contracts to store something besides `result: String` (as discussed in oracle)

Reply:
removed from apis and returned back to `verifier_simple` contract's state in 5afcc91ecbd7